contracts/Pool.sol

```diff
@@ -130,8 +126,10 @@ contract Pool is PoolFDT {
         liquidityCap = _liquidityCap;
 
         // Initialize the LiquidityLocker and StakeLocker
-        stakeLocker     = address(IStakeLockerFactory(_slFactory).newLocker(_stakeAsset, _liquidityAsset));
-        liquidityLocker = address(ILiquidityLockerFactory(_llFactory).newLocker(_liquidityAsset));
+        stakeLocker              = address(IStakeLockerFactory(_slFactory).newLocker(_stakeAsset, _liquidityAsset));
+        address _liquidityLocker = address(ILiquidityLockerFactory(_llFactory).newLocker(_liquidityAsset));
+        liquidityLocker          = _liquidityLocker;
+        fundsToken               = IERC20(_liquidityLocker);
```

Comment:
Unlike for Loan and StakeLocker, where the "fundsToken" is known at the start of the constructor, and can be passed to the super constructor, here it is a newly deployed contract, and thus only known later. That means that `fundsToken` for the Pool is not immutable, and thus a bit ore costly. If anyone know a better way to do this, please opine.